### PR TITLE
Implement #107 - move fetch button and change local status message

### LINKED ISSUE #107: Move fetch button and change local status messge
Continuation of #88, the following functionalities were requested:
- move the fetch button from the footer to the header
- change the text of the local status to indicate requiring a fetch in order to update with the remote
- make local status display both ahead and behind if local repo is both, instead of just behind

### PR DESCRIPTION
**Pull request by ajlarivi**
closes #107

also made the local status display both ahead and behind in the case that both are true

## test cases:

**fetch button works**
- open a repository in visual git
- make a change on the remote repo
- click the fetch button
- "your branch is 1 behind remote" is displayed

**both ahead and behind are displayed**
- open a repository in visual git
- make a change locally and commit it
- make a change on the remote and commit it
- click the fetch button
- both "your branch is 1 behind remote" and "your branch is 1 ahead of remote" are displayed

## WIP styling
- any suggestions for the fetch icon or the styling of the local status div in the bottom right are welcome, currently not really happy with either

### CODE REVIEW

**Comment by ajlarivi** [comment]
**note**: branch currently not rebased with master, still on old version of node and electron

**Comment by ajlarivi** [comment]
branch rebased with master, ready for reviews

**Review by smithmr — CHANGES_REQUESTED** [review]
## Test failed
- Made commit on remote branch through github.
- went back to visualgit and on the same branch, I clicked fetch. It says I am up to date. If I click push it says I am behind remote by 1 commit.

**Comment by cchampernowne** [comment]
Does the code work? 
-yes
Did I test it locally? 
-Yes
What OS am I using? 
-Ubuntu 3.16
Does it perform its intended function, the logic is correct etc.
-Yes. I tested both a local change, to cause my branch to be ahead, and tested remote changes, causing my local branch to be behind
Is the code well documented? Would it be easily understood to other developers?
-Yes, comments are included where needed
Does it have correct linting/formatting?
-Yes
Is there any redundant or duplicate code? Any hard-coded values?
-not that I saw
Can any of the code be replaced with library functions?
-N/A
Are good names used for methods, variables, etc.? Do they follow naming conventions?
-Yes
Were any other issues found?
-Not related to this issue.

**Review by cchampernowne — APPROVED** [review]
Does the code work? 
-yes
Did I test it locally? 
-Yes
What OS am I using? 
-Ubuntu 3.16
Does it perform its intended function, the logic is correct etc.
-Yes. I tested both a local change, to cause my branch to be ahead, and tested remote changes, causing my local branch to be behind
Is the code well documented? Would it be easily understood to other developers?
-Yes, comments are included where needed
Does it have correct linting/formatting?
-Yes
Is there any redundant or duplicate code? Any hard-coded values?
-not that I saw
Can any of the code be replaced with library functions?
-N/A
Are good names used for methods, variables, etc.? Do they follow naming conventions?
-Yes
Were any other issues found?
-Not related to this issue.

**Comment by Leon0315** [comment]
> ## Test failed
> * Made commit on remote branch through github.
> * went back to visualgit and on the same branch, I clicked fetch. It says I am up to date. If I click push it says I am behind remote by 1 commit.

Had the same issue with Matt, the fetch button failed to update the message

**Review by jbenning97 — APPROVED** [review]
**Does the code work? Did I test it locally? What OS am I using? Does it perform its intended function, the logic is correct etc.** - Yes, the code works for me. Testing on macOS 10.13.

**Is the code well documented? Would it be easily understood to other developers?** 

Yes

**Does it have correct linting/formatting?**

 Yes

**Is there any redundant or duplicate code? Any hard-coded values?**

 No, there were not redundant or duplicate code. 

**Can any of the code be replaced with library functions?** 

No

**Are good names used for methods, variables, etc.? Do they follow naming conventions?** 

Yes

**Were any other issues found?** 

I did not find any other issues during my testing.

File: app/misc/git.ts [hunk 1]
```diff
@@ -1124,23 +1124,30 @@ function checkIfExistOrigin(branchName) {
 //calls getAheadBedhindCommits to display status of local repo to user
 function displayAheadBehind() {
   let branch = document.getElementById("branch-name").innerText;
```
Comment (smithmr) [inline app/misc/git.ts]: Use the getBranch() function here

**Review by smithmr — CHANGES_REQUESTED** [review]
## Tested and passed 
- tested on macbook pro
- well documented
- works as intended
## Quick fixes
- use getBranch() instead of calling the dom to get the branch name.

**Review by smithmr — APPROVED** [review]

**Pull request closed without merging**